### PR TITLE
Drop view and or replace added to exporting view #14404

### LINKED ISSUE #14404: "DROP VIEW" and "IF NOT EXISTS" checkboxes have no effect when exporting views
### Steps to reproduce
1. Navigate to the Export page of a view
2. Change the export method from quick to custom
3. Activate the checkboxes for "`DROP VIEW`" and/or "`IF NOT EXISTS`"

### Expected behaviour
1. I expect a `DROP VIEW` query prior to the `CREATE` query when activating the `DROP VIEW` checkbox.
2. I did not expect an "`IF NOT EXISTS`" checkbox as this seems impossible when creating a view.

### Actual behaviour
There is no difference in the output no matter if these checkboxes are checked or not.

### Suggestion for improvement
Replace the "`IF NOT EXISTS`" checkbox by something like a "`REPLACE`" checkbox in order to achieve an output query like "`CREATE OR REPLACE VIEW view_name"

### Server configuration
**Operating system**:
Windows 7

**Web server:**
Apache 2.4.16

**Database:**
MySQL 5.6.17

**PHP version:**
5.6.12

**phpMyAdmin version:**
Tested with 5.7.7 and 5.8.1

### PR DESCRIPTION
**Pull request by aswaniprakash**
Has depenndancy in the CreateStatement.php file in sql-parser
PR: https://github.com/phpmyadmin/sql-parser/pull/198

Signed-off-by: Aswani Prakash <aswani15prakash@gmail.com>

Fixes #14404

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

File: libraries/classes/Plugins/Export/ExportSql.php [hunk 1]
```diff
@@ -305,11 +305,21 @@ protected function setProperties()
...
+                $subgroup_create_view->setSubgroupHeader($leaf);
+
+                $leaf = new BoolPropertyItem(
+                    'or_replace_view',
+                    sprintf(__('%s view'), '<code>OR REPLACE</code>')
+                );
+                $subgroup_create_view->addProperty($leaf);
+
+                $subgroup->addProperty($subgroup_create_view);
+
```
Comment (mauriciofauth) [inline libraries/classes/Plugins/Export/ExportSql.php]: Please remove this blank line. See: https://travis-ci.org/phpmyadmin/phpmyadmin/jobs/399466397#L733

File: libraries/classes/Plugins/Export/ExportSql.php [hunk 2]
```diff
@@ -1462,6 +1472,15 @@ public function getTableDef(
...
 
+        if (!empty($sql_drop_table)
+            && $GLOBALS['dbi']->getTable($db, $table)->isView()
+        ) {
+            $schema_create .= 'DROP VIEW IF EXISTS '
+                . Util::backquote($table_alias, $sql_backquotes) . ';'
+                . $crlf;
+        }
+
+
```
Comment (mauriciofauth) [inline libraries/classes/Plugins/Export/ExportSql.php]: Please remove this blank line. See: https://travis-ci.org/phpmyadmin/phpmyadmin/jobs/399466397#L736

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Hello, thank you for this contribution.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Tests for the PhpMyAdmin\Plugins\Export\ExportSql class failed.
See: https://travis-ci.org/phpmyadmin/phpmyadmin/jobs/438975086#L722

**Comment by mauriciofauth** [comment]
Merged in cd752ad149410f4778cb0a60539f0856733c029b, thanks for your contribution!

**Pull request closed without merging**